crystal/distribution.yaml

```diff
@@ -813,7 +813,7 @@ repositories:
       url: https://github.com/ros2-gbp/rcl-release.git
       version: 0.6.2-0
     source:
-      test_pull_requests: true
+      test_commits: false
```

Comment:
http://www.ros.org/reps/rep-0143.html has both test_pull_requests and test_commits. What's the difference?

```
test_commits: a boolean flag used to enable CI jobs for each commit to the branch specified under versions. (default: false)
test_pull_requests: a boolean flag used to enable CI jobs for each pull request against the branch specified under version. (default: false)
```

Reply:
One tests each commit to the source branch, the other tests each pull request targeting the source branch.

Reply:
My information is based on observation rather than specification
test_commits runs a job whenever a new commit is made to a specific branch.
test_pull_requests runs a new job whenever a PR is opened against a branch.